app/src/visionglass/res/values/themes.xml

```diff
@@ -15,6 +15,156 @@
+        <item name="iconGravity">textStart</item>
+        <item name="iconPadding">0dp</item>
+        <item name="iconTint">@color/white</item>
+        <item name="shapeAppearanceOverlay">@style/ShapeAppearanceOverlay.App.Button.Rounded</item>"
+    </style>
+
+    <style name="BaseFooterButton" parent="Widget.MaterialComponents.Button">
+        <item name="android:layout_width">wrap_content</item>
+        <item name="android:layout_height">?attr/actionBarSize</item>
+        <item name="android:scaleType">fitCenter</item>
+        <item name="android:backgroundTint">@color/asphalt</item>
+        <item name="iconGravity">textStart</item>
+        <item name="iconPadding">0dp</item>
+        <item name="iconTint">@color/white</item>
+        <item name="shapeAppearanceOverlay">@style/ShapeAppearanceOverlay.App.Button.Rounded</item>"
+    </style>
+
+    <style name="BaseBorderlessButton" parent="Widget.AppCompat.Button.Borderless">
+        <item name="android:layout_width">0dp</item>
+        <item name="android:layout_height">match_parent</item>
+        <item name="iconGravity">textStart</item>
+        <item name="iconPadding">0dp</item>
+        <item name="iconTint">@color/white</item>
+    </style>
+
+    <style name="BaseMediaButton" parent="BaseBorderlessButton">
+        <item name="android:layout_weight">1</item>
+    </style>
+
+
+    <!-- Specific Styles -->
+    <style name="ToolbarStyle" parent="Widget.MaterialComponents.Toolbar.Primary">
+        <item name="android:layout_width">match_parent</item>
+        <item name="android:layout_height">?attr/actionBarSize</item>
+        <item name="android:elevation">0dp</item>
+        <item name="android:paddingStart">8dp</item>
+        <item name="android:paddingEnd">8dp</item>
+        <item name="logoAdjustViewBounds">true</item>
+        <item name="navigationIcon">@drawable/ff_logo_icon_48</item>
+        <item name="popupTheme">@style/Theme.WolvicPhone.PopupOverlay</item>
+    </style>
+
+    <style name="HeadlockToggleButtonStyle" parent="BaseToolbarButton">
+        <item name="icon">@drawable/headlock_toggle_button_states</item>
+        <item name="android:checkable">true</item>
+        <item name="android:checked">false</item>
+        <item name="layout_constraintEnd_toStartOf">@id/realign_button</item>
+        <item name="layout_constraintStart_toStartOf">parent</item>
+        <item name="layout_constraintTop_toTopOf">parent</item>
+    </style>
+
+    <style name="RealignButtonStyle" parent="BaseToolbarButton">
+        <item name="layout_constraintEnd_toEndOf">@id/voice_search_button</item>
+        <item name="layout_constraintStart_toEndOf">@id/headlock_toggle_button</item>
+        <item name="layout_constraintTop_toTopOf">parent</item>
+    </style>
+
+    <style name="VoiceSearchButtonStyle" parent="BaseToolbarButton">
+        <item name="icon">@drawable/ic_icon_microphone</item>
+        <item name="android:checkable">true</item>
```

Comment:
It was like this before, but I think that it makes more sense to set use

    <item name="android:checkable">false</item>

because we don't have a way to update this UI when the user dismisses the voice search.